skimage/io/tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+import numpy as np
+import imageio.v3 as iio
+
+
+@pytest.fixture(scope="session")
+def random_gif_path(tmp_path_factory):
```

Comment:
This fixture does not appear to work on Windows.

Reply:
https://dev.azure.com/scikit-image/scikit-image/_build/results?buildId=9685&view=logs&j=97a3063c-824d-5691-874a-4213ba287284&t=d55d7a31-a919-5bb6-273c-a812138f063e&l=8484

Reply:
The fixture does what it claims (it produces a gif with frames of shape `(25,01,3)`); however, the result is compared against an expected shape of `(10, 10, 3)`. Is there some preprocessing in the `imgs` fixture that is not working in the expected way?